org.kde.kdenlive.json

```diff
@@ -0,0 +1,466 @@
+                    "dest-filename": "gtest.tar.gz"
+                },
+                {
+                    "type": "shell",
+                    "commands": [
+                        "tar xf gtest.tar.gz --strip 1"
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "libsamplerate",
+            "config-opts": [
+                "--disable-static"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://www.mega-nerd.com/SRC/libsamplerate-0.1.9.tar.gz",
+                    "sha256": "0a7eb168e2f21353fb6d84da152e4512126f7dc48ccb0be80578c565413444c1"
+                }
+            ]
+        },
+        {
+            "name": "sox",
+            "config-opts": [
+                "--disable-static"
+            ],
+            "cleanup": [
+                "/bin"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://sourceforge.net/projects/sox/files/sox/14.4.2/sox-14.4.2.tar.bz2",
+                    "sha256": "81a6956d4330e75b5827316e44ae381e6f1e8928003c6aa45896da9041ea149c"
+                }
+            ]
+        },
+        {
+            "name": "vidstab",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/georgmartius/vid.stab/archive/v1.1.0.tar.gz",
+                    "sha256": "14d2a053e56edad4f397be0cb3ef8eb1ec3150404ce99a426c4eb641861dc0bb"
+                }
+            ]
+        },
+        {
+            "name": "openjpeg",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release"
+            ],
+            "cleanup": [
+                "/bin",
+                "/lib/openjpeg-*",
+                "*.a"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/uclouvain/openjpeg/archive/v2.3.0.tar.gz",
+                    "sha256": "3dc787c1bb6023ba846c2a0d9b1f6e179f1cd255172bde9eb75b01f1e6c7d71a"
+                }
+            ]
+        },
+        {
+            "name": "x264",
+            "config-opts": [
+                "--disable-cli",
+                "--enable-shared"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.videolan.org/x264/snapshots/x264-snapshot-20171223-2245-stable.tar.bz2",
+                    "sha256": "ee4ba2c3d1caf786ad66d3fa86bdc368a8f827318788a62a9c8444b2c35816bf"
+                }
+            ]
+        },
+        {
+            "name": "x265",
```

Comment:
In order to build on arm and aarch64 you're going to need to replicate the fixes here: https://github.com/flathub/org.pitivi.Pitivi/commit/548615c8a5d389197438997ce63d324d5c4b17b5